src/models/monitoringStandard.js

```diff
@@ -17,7 +17,7 @@ export default (sequelize, DataTypes) => {
         models.MonitoringStandard,
         {
           foreignKey: 'standardId',
-          as: 'monitoringStandardes',
```

Comment:
yikes

Reply:
This looks like it has also already been fixed.

Reply:
Can't be lowering our Standardes must remain vigilant lol